src/js/components/agencyV2/overview/FySummary.jsx

```diff
@@ -30,8 +76,15 @@ const FySummary = ({ isMobile, fy }) => {
                 subtitle={isMobile ? 'How much can this agency spend?' : (<>How much can<br />this agency spend?</>)}
                 data={totalBudgetaryResources}
                 secondaryData={`${percentOfFederalBudget} of the FY ${fy} U.S. federal budget`}
-                label="Total Budgetary Resources Over Time" >
-                <ComingSoon className="viz-placeholder" />
+                label="Total Budgetary Resources Over Time">
+                {isLoading && <LoadingMessage />}
```

Comment:
Should loading and error messaging along with rendering of the chart be happening in this array of components?

It almost seems like a difference of concerns almost, where this array should only be a data mapping and the actual visualization should be deciding what to render?

Reply:
Good call. Moved the logic to the BarChart component.